src/patternfly/sass-utilities/mixins.scss

```diff
@@ -309,3 +309,28 @@
     }
   }
 }
+
+// Applies custom styles to support RTL
+@mixin pf-v5-rtl($transform: null) {
+  // resets property used to inverse transform values calc()s in RTL
+  @if $transform == "transform-flip-revert" {
```

Comment:
Only nit is the naming of this particular value. Just wondering if "transform-flip(-revert)" conveys exactly what it's for, or if it's too close to "flip-inline"... "transform-modifier(-revert)" or "transform-multiplier(-revert)" maybe?